gui/widgets/matRad_DVHStatsWidget.m

```diff
@@ -1,54 +1,102 @@
                     'Colormap',[0 0 0.5625;0 0 0.625;0 0 0.6875;0 0 0.75;0 0 0.8125;0 0 0.875;0 0 0.9375;0 0 1;0 0.0625 1;0 0.125 1;0 0.1875 1;0 0.25 1;0 0.3125 1;0 0.375 1;0 0.4375 1;0 0.5 1;0 0.5625 1;0 0.625 1;0 0.6875 1;0 0.75 1;0 0.8125 1;0 0.875 1;0 0.9375 1;0 1 1;0.0625 1 1;0.125 1 0.9375;0.1875 1 0.875;0.25 1 0.8125;0.3125 1 0.75;0.375 1 0.6875;0.4375 1 0.625;0.5 1 0.5625;0.5625 1 0.5;0.625 1 0.4375;0.6875 1 0.375;0.75 1 0.3125;0.8125 1 0.25;0.875 1 0.1875;0.9375 1 0.125;1 1 0.0625;1 1 0;1 0.9375 0;1 0.875 0;1 0.8125 0;1 0.75 0;1 0.6875 0;1 0.625 0;1 0.5625 0;1 0.5 0;1 0.4375 0;1 0.375 0;1 0.3125 0;1 0.25 0;1 0.1875 0;1 0.125 0;1 0.0625 0;1 0 0;0.9375 0 0;0.875 0 0;0.8125 0 0;0.75 0 0;0.6875 0 0;0.625 0 0;0.5625 0 0],...
-                    'MenuBar','none',...
+                    'MenuBar','figure',...
+                    'ToolBar','figure',...
                     'Name','MatRad Plan Analysis',...
                     'NumberTitle','off',...
                     'HandleVisibility','callback',...
-                    'Tag','figure1');
+                    'Tag','figDVHStat');
                 
             end
-            
-            this = this@matRad_Widget(handleParent);   
+            this = this@matRad_Widget(handleParent);
+            this.SelectedDisplayOption = SelectedDisplayOption;
+            this.dvhWidgetHandle.SelectedCube = SelectedDisplayOption;
+            this.statWidgetHandle.SelectedCube = SelectedDisplayOption;
+            this.lockUpdate = true;
+            this.dvhWidgetHandle.lockUpdate = true;
+            this.statWidgetHandle.lockUpdate = true;
+            this.update();
+
         end        
         
         function this=update(this,evt)
-            if nargin == 2
-                doUpdate = this.checkUpdateNecessary({'resultGUI','cst','pln'},evt);
-                
-                if doUpdate
-                    this.dvhWidgetHandle.update(evt);
-                    this.statWidgetHandle.update(evt);
+            if this.lockUpdate
+                if nargin == 2
+                    doUpdate = this.checkUpdateNecessary({'resultGUI','cst','pln'},evt);
+
+                    if doUpdate
+                        this.dvhWidgetHandle.update(evt);
+                        this.statWidgetHandle.update(evt);
+                    end
+                else
+                    if    ~strcmp(this.dvhWidgetHandle.SelectedCube, this.SelectedDisplayOption)
+                        this.dvhWidgetHandle.SelectedCube = this.SelectedDisplayOption;
+                        this.statWidgetHandle.SelectedCube = this.SelectedDisplayOption;
+
+                    end
+                        this.dvhWidgetHandle = this.dvhWidgetHandle.update();
+                        this.statWidgetHandle = this.statWidgetHandle.update();
+                        if numel(this.dvhWidgetHandle.widgetHandle.Children)>2 
+                            this.removeOverlap();
+                        end
```

Comment:
indentation of lines 80-84 is wrong, whitespaces around ' > 2' in line 82 need adjustment